# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../acx100/eth-naming.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -ur a/pci.c a-unified/pci.c
--- a/pci.c	2005-10-02 12:57:19.000000000 +0200
+++ a-unified/pci.c	2006-06-06 17:42:17.000000000 +0200
@@ -1964,7 +1964,7 @@
 		goto fail_irq;
 	}
 
-	dev = alloc_netdev(sizeof(wlandevice_t), "wlan%d", acx_netdev_init);
+	dev = alloc_netdev(sizeof(wlandevice_t), "eth%d", acx_netdev_init);
 	/* (NB: memsets to 0 entire area) */
 	if (!dev) {
 		printk("acx: no memory for netdevice structure\n");
diff -ur a/usb.c a-unified/usb.c
--- a/usb.c	2005-10-02 12:57:19.000000000 +0200
+++ a-unified/usb.c	2006-06-06 17:42:02.000000000 +0200
@@ -487,7 +487,7 @@
 	/* Ok, so it's our device and it is already booted */
 
 	/* Allocate memory for a network device */
-	dev = alloc_netdev(sizeof(wlandevice_t), "wlan%d", acx_netdev_init);
+	dev = alloc_netdev(sizeof(wlandevice_t), "eth%d", acx_netdev_init);
 	/* (NB: memsets to 0 entire area) */
 	if (!dev) {
 		msg = "acx: no memory for netdev\n";
